src/main/webapp/recommendation-map.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Student Recommendations</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script src="/scripts/map.js"></script>
+    <script src="/scripts/common.js"></script>
```

Comment:
I don't think we have a file with that name? 🤔

Reply:
yup not before, but I just renamed the file based on Kazi's review suggestion

Reply:
index.js is now renamed to common.js since it contains the function for scroll up button which will be used in most pages

Reply:
sounds good! Thanks :)